notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>I think we could make a really cool comparative seasonal load chart here with <code>seaborn</code>  -- with a grid of different states and a statistical envelope showing the monthly load for each state across all of the years of data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>ooo that would be cool!</p><p> </p>